dvclive/lightning.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, Optional, Union
+
+from pytorch_lightning.loggers import LightningLoggerBase
+from pytorch_lightning.loggers.base import rank_zero_experiment
+from pytorch_lightning.utilities import rank_zero_only
+from torch import is_tensor
+
+import dvclive
+
+
+class DvcLiveLogger(LightningLoggerBase):
+    def __init__(
+        self,
+        run_name: Union[str, int],
```

Comment:
What is this used for? 

I see that it is used for the `version` property, just wondering what Pytorch Lightning uses that property for  🤔

Reply:
Usually it is the name of the run, that can be seen in the UI of logger.
Inside Pytorch Lightning I've seen the use of this property only in a method that form data for progress bar. But I have never seen any version numbers in the progress bar itself in practice

Reply:
Thanks for the explanation. 

TBH the property kind of feels useless in DVCLive (+ DVC) context as DVC already generates a unique experiment name for each run. 

I wonder if we could just make it optional with a reasonable default ("dvclive_run"?)?

Reply:
I agree, it will be useless in most cases. Made it optional with "dvclive_run" default